scripts/triggers-end-to-end-tests/tests.ts

```diff
@@ -295,6 +295,54 @@ describe("storage emulator function triggers", () => {
     expect(test.storageDeletedTriggerCount).to.equal(1);
     expect(test.storageV2DeletedTriggerCount).to.equal(1);
   });
+
+  it("should write to a specific bucket in the storage emulator", async function (this) {
+    this.timeout(EMULATOR_TEST_TIMEOUT);
+
+    const response = await test.writeToSpecificStorageBucket();
+    expect(response.status).to.equal(200);
+    await new Promise((resolve) => setTimeout(resolve, EMULATORS_WRITE_DELAY_MS));
+  });
+
+  it("should have triggered cloud functions", () => {
+    // default bucket values
+    expect(test.storageFinalizedTriggerCount).to.equal(2);
```

Comment:
q: it should've not incremented these counts right? Are we just checking here that this values as same as it was before?

(in that case, wow we should update the testing harness to clear the counts between test cases)

Reply:
Yeah I was just checking the values from the previous run, I added in a `resetCounts` function in the framework to clean this up a bit